src/Compilers/CSharp/Portable/Binder/Binder_Operators.cs

```diff
@@ -2138,6 +2138,7 @@ private BoundExpression BindAddressOfExpression(PrefixUnaryExpressionSyntax node
         // Basically a port of ExpressionBinder::isFixedExpression, which basically implements spec section 18.3.
         // Renamed because there are already too many things called "fixed".
         // NOTE: internal purely for testing purposes.
+        // CONSIDER: why is it used in product code? also, accessedLocalOrParameterOpt is not used anywhere!
```

Comment:
>// CONSIDER: why is it used in product code? also, accessedLocalOrParameterOpt is not used anywhere! [](start = 8, length = 100)

I think the comment above says that it is not private because we want to be able to use it in unit-tests. Form that perspective, it is fine to use use this function in production code.

Reply:
Thanks for clarifying. Will remove that.

---
In reply to: [194577195](https://github.com/dotnet/roslyn/pull/27424#discussion_r194577195) [](ancestors = 194577195)